### DISCUSSION TITLE
How do I login in code-server Docker?

### DISCUSSION THREAD

**Discussion by gerroon** [aws/aws-toolkit-vscode#5740] [Q&A]
Hi

I installed it under code-server Docker. I installed the extension. I can't get the extension to bring the login screen thingy so that I can login to Amazon with my account. I used it on regular Vscode so I know how easy it is to install and run but there is no popup or secondary window on code-server

I am stuck here. Is there another way to login without needing this type of interaction?

![image](https://github.com/user-attachments/assets/1a28e013-4e8d-4012-a611-9adbe11b728e)

**Comment by justinmk3** [comment]
Normally the extension should detect if it's running in a remote context and then switch to the "device code" auth flow. But it sounds like maybe that's not working. Please consider opening an issue with logs and details about your system/versions etc.

**Reply by gerroon** [reply]
Quick question, does it use the connections over the codeserver's Docker container or does it use the connectivity of the web browser that is access code-server?

**Reply by justinmk3** [reply]
vscode extensions don't have access to the browser or its cookies. They run in the remote container.
